spec/spec.md

```diff
@@ -80,21 +80,27 @@ DELETE <ECHO_SERVER_URL>/clients/<CLIENT_ID>
 
 ### Send Notification
 
-> **Note**
-> This will have E2E encryption in the future but for now is designed without encryption.
-
 ```
 POST <ECHO_SERVER_URL>/clients/<CLIENT_ID>
 {
     "id": "0000-0000-0000-0000"
     "payload": {
-        "title": "New Request",
-        "description": "You have a signing request."
+      "topic": "<topic>",
```

Comment:
Should we explain what the topic is (the hash of blah blah blah...)?

Reply:
Thats okay, this is going to be moved into the main specs at some point and we can link it then